homeassistant/components/light/device_automation.py

```diff
@@ -0,0 +1,74 @@
+"""Provides device automations for lights."""
+import voluptuous as vol
+
+import homeassistant.components.automation.state as state
+from homeassistant.core import split_entity_id
+from homeassistant.const import (
+    CONF_DEVICE_ID, CONF_DOMAIN, CONF_ENTITY_ID, CONF_PLATFORM, CONF_TYPE)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity_registry import async_entries_for_device
+from . import DOMAIN
+
+ENTITY_TRIGGERS = [
+    {
+        # Trigger when light is turned on
+        CONF_PLATFORM: 'device',
+        CONF_DOMAIN: DOMAIN,
+        CONF_TYPE: 'turn_on',
```

Comment:
let's create constants for `turn_on` and `turn_off`

Reply:
Fixed.